tests/test_helpers.py

```diff
@@ -32,6 +32,27 @@
       "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
       "--title", "On the roundness of the Earth", "--status", "in-prep"],
      "2006as_mars has been added in presentations\n"),
+    (["helper", "l_progress", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+     ),
+    (["helper", "l_progress", "-v", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+      "    log_url: https://docs.google.com/document/d/1YC_wtW5Q\n"
+      "    team:\n"
+      "        lead: ascopatz\n"
+      "        group_members: ascopatz\n"
+      "        collaborators: aeinstein, pdirac\n"
+      "    milestones:\n"
```

Comment:
The most important milestone is the deliverable.  Let's have a brief summary of the deliverable.  Maybe it could be something like:
```
deliverable:
       f"type {prum['deliverable'].get('type','unknown')} will be delivered on {prum['deliverable'].get('due_date')} to "
       f"{prum['deliverable'].get('target_audience')} covering scope {prum['deliverable'].get('scope')}"
```
where turn the scope list into a comma-separated string.

Reply:
I add the deliverable outside milestones. I checked our database and found out that the scope of the deliverable is usually a long list containing many elements. So I switched to the following format, which I think maybe more organized.

deliverable:
    due_date: 2021-05-05
    audience: beginning grad in chemistry
    scope: 1. UCs that are supported or some other scope description if it is software
                2. sketch of science story if it is paper
    platform: description of how and where the audience will access the deliverable.  Journal if it is a paper

I picked the due_date, audience, scope and the platform, which I find most important. Let me know if you want to change the format or if you want to include more information.